diff --git a/node_modules/@react-navigation/elements/src/Screen.tsx b/node_modules/@react-navigation/elements/src/Screen.tsx
index 6ea2597..06f6f50 100644
--- a/node_modules/@react-navigation/elements/src/Screen.tsx
+++ b/node_modules/@react-navigation/elements/src/Screen.tsx
@@ -81,6 +81,7 @@ export default function Screen(props: Props) {
                 setHeaderHeight(height);
               }}
               style={headerTransparent ? styles.absolute : null}
+              pointerEvents={headerTransparent ? 'box-none': 'auto'}
             >
               {header}
             </View>
